packages/app-store/stripepayment/lib/utils.ts

```
@@ -17,6 +18,14 @@ export function getPerSeatPlanPrice(): string {
   return STRIPE_TEAM_MONTHLY_PRICE_ID;
 }
 
+export function getPhoneNumberMonthlyPriceId(): string {
+  return STRIPE_PHONE_NUMBER_MONTHLY_PRICE_ID;
+  if (!STRIPE_PHONE_NUMBER_MONTHLY_PRICE_ID) {
+    throw new Error("STRIPE_PHONE_NUMBER_MONTHLY_PRICE_ID env var is not set");
+  }
+  return STRIPE_PHONE_NUMBER_MONTHLY_PRICE_ID;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)